fastats/core/windowed_stateful_pass.py

```diff
@@ -4,8 +4,8 @@
 from fastats.core.decorator import fs
 
 
-def value(x):
-    return x[0]
+def value(x, val_in, val_out, state):
```

Comment:
Moved to separate module so there's no conflicts for the `value` function in #22